mkdocs.yml

```
@@ -114,6 +114,7 @@ nav:
       - 'PHP API': 'migration/wsc54/php.md'
       - 'TypeScript and JavaScript': 'migration/wsc54/javascript.md'
       - 'Third Party Libraries': 'migration/wsc54/libraries.md'
+      - 'Templates': 'migration/wsc54/templates.md'
```

```suggestion
      - 'TypeScript and JavaScript': 'migration/wsc54/javascript.md'
      - 'Templates': 'migration/wsc54/templates.md'
      - 'Third Party Libraries': 'migration/wsc54/libraries.md'
```
I'd interchange the two, because third party libraries is not directly related to our code and it's consistent with 5.3.